runtime/vm/intfunc.c

```diff
@@ -21,6 +21,7 @@
  *******************************************************************************/
 
 #include "j9protos.h"
+#include "vm_api.h"
```

Comment:
vm_api.h is included in j9protos.h, you should be able to compile without adding this line